dashboard/src/features/accounts/components/AccountUpdate.jsx

```diff
@@ -0,0 +1,119 @@
+import React from 'react'
+import { BaseUpdate, FormContainer, FormSection, JsonField, NotFound } from 'features/shared/components'
+import { reduxForm } from 'redux-form'
+
+class Form extends React.Component {
+  constructor(props) {
+    super(props)
+
+    this.submitWithErrors = this.submitWithErrors.bind(this)
+
+    this.state = {}
+  }
+
+  submitWithErrors(data) {
+    return new Promise((resolve, reject) => {
```

Comment:
I realize this is copy/paste of other code, but as a re-read it, I wonder if there's a simpler pattern here:

```
submitWithErrors(data) {
  return this.props.submitForm(data, this.props.item.id).catch(err => {
    throw {_error: err}
  })
}
```